scripts/fast-reboot

```diff
@@ -426,7 +426,7 @@ if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
 
     FILTER_FDB_ENTRIES_RC=0
     # Filter FDB entries using MAC addresses from ARP table
-    /usr/bin/filter_fdb_entries.py -f $DUMP_DIR/fdb.json -a $DUMP_DIR/arp.json || FILTER_FDB_ENTRIES_RC=$?
+    /usr/bin/filter_fdb_entries.py -f $DUMP_DIR/fdb.json -a $DUMP_DIR/arp.json -c $DUMP_DIR/config_db.json || FILTER_FDB_ENTRIES_RC=$?
```

Comment:
is it guaranteed to have config_db in $DUMP_DIR location?

Reply:
I doubt we will have config_db.json there. Can you use /etc/sonic/config_db.json?

Reply:
Thanks! I missed that DUMP_DIR is being created a couple of lines above! Will update